packages/nds-checkbox/README.md

```diff
@@ -0,0 +1,134 @@
+# `@nice-digital/nds-checkbox`
+
+> Checkbox component for the NICE Design System
+
+- [`@nice-digital/nds-checkbox`](#nice-digitalcheckbox) - [Installation](#installation) - [Usage](#usage) - [React](#react) - [Props](#props) - [SCSS](#scss) - [HTML](#html)
+
+## Installation
+
+Install Node, and then:
+
+```sh
+npm i @nice-digital/nds-checkbox --save
+```
+
+## Usage
+
+### React
+
+Import the `Checkbox` component from the package and use within JSX:
+
+```jsx
+import React from "react";
+import { Checkbox } from "@nice-digital/nds-checkbox";
+
+// The mimimum to be supplied for the input to render is a value and a name
+
+<Checkbox value="yes" name="contact-permission" />;
+```
+
+> Note: The React component automatically imports the SCSS, so there's no need to import the SCSS directly yourself.
+
+#### Props
+
+##### value
+
+- Type: `string` (required)
+
+The value for the single input.
+
+```js
+<Checkbox value="yes" name="contact-permission" />
+```
+
+##### name
+
+- Type: `string` (required)
+
+The name attribute on the single input
+
+```js
+<Checkbox label="yes" name="contact-permission" />
+```
+
+##### label
+
+- Type: `string`
+
+The label for the single input.
+
+```js
+<Checkbox label="yes" name="contact-permission" label="Yes, please." />
+```
+
+##### disabled
+
+- Type: `boolean`
+
+Prop to control whether the disabled attribute is present on the input.
+
+```js
+<Checkbox label="yes" name="contact-permission" disabled={true} />
+```
+
+##### error
```

Comment:
Can we use `error=string` instead? Then we can use an error message e.g. https://design-system.service.gov.uk/components/text-input/#error-messages for accessibility

Reply:
Yep that makes sense - will do the same for radio

Reply:
@ediblecode [Looking at GOV](https://design-system.service.gov.uk/components/checkboxes/error/index.html), they have an error state that covers the group rather than an individual checkbox component. Do you think there's a place for an error message on a per-checkbox?

Reply:
I think so - maybe for a single checkbox that you're required to check for example terms and conditions?